### DISCUSSION TITLE
Is there a single step debugger for zeptoforth?

### DISCUSSION THREAD

**Discussion by marklacas** [tabemann/zeptoforth#21] [Q&A] [answered]
pForth has a very nice single step debugger that I find very useful.  Is there anything like that in ZeptoForth?

**Comment by tabemann** [comment, marked as answer]
zeptoforth itself does not come with a single-step debugger, but you can use it with gdb via OpenOCD and SWD; note that zeptoforth is distributed with ELF files for the kernel, which can be used with gdb.
